config/features.rb

```diff
@@ -28,4 +28,8 @@
   feature :notes_tab_form,
           default: false,
           description: "Enables the Notes tab in the deposit form"
+
+  feature :cache_enabled,
+          default: false,
+          description: "TURBO MODE. Turns on cache for API endpoints. Experimental"
```

Comment:
Can you remove the "TURBO MODE.", its not necessary.

Reply:
Spoilsport :P